src/bundles/comms.stanza

```diff
@@ -311,7 +311,51 @@ specification is present in NXP's references
 @member ws Word Select - typically for left vs right differentiation
 @member sd Data Signal
 <DOC>
-public pcb-bundle i2s :
+
+public pcb-enum jsl/bundles/comms/I2SPins :
+  I2S-MCK 
```

Comment:
`MCK` is not part of the protocol specification. There is no reason it must come from the microcontroller. It could just as easily come from a standalone oscillator.

I don't think this should be included.